app/controllers/credentials.controller.js

```diff
@@ -187,9 +187,14 @@ async function updateNotificationCredentials (req, res, next) {
   }
 }
 
+function checkOrgDetails (req, res, next) {
```

Comment:
This should be under stripe-setup as check organisation details is part of stripe onboarding

Reply:
Thanks - created a new controller